### PR TITLE
chore: move to yarn workspaces + introduce nx

### PR DESCRIPTION
**Pull request by NuttyShrimp**
Moved the dependency manager to yarn so we can use the workspace feature (could also be available in npm but yarn is faster overall)
I've also added [nx](https://nx.dev/) it is a monorepo management tool which allows us to run 1 `yarn dev` or `yarn build` cmd to build/watch loama + all the libraries it depends on in this repo

### CODE REVIEW

**Comment by bjdmeest** [comment]
Can you update the README(s) so I know what to do to get everything running?

**Comment by bjdmeest** [comment]
And the CHANGELOG too

File: loama/.env
Comment (bjdmeest) [inline loama/.env:1]: I don't understand why you remove this .env file. If you don't like to commit .env files, at least rename to `.env.example`

Comment (NuttyShrimp) [reply loama/.env:1]: There is a .env.example file present in the project https://github.com/NuttyShrimp/loama/blob/chore/add-workspaces/loama/.env.example. Hence, why I removed the "configured" one

Comment (bjdmeest) [reply loama/.env:1]: ah, check!

**Review by bjdmeest — APPROVED** [review]

**Pull request merged**